rep-2000.rst

```diff
@@ -1179,6 +1179,120 @@ Package manager use for dependencies:
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gazebo         |     Ionic*     |      N/A       |   N/A    |    Ionic*     |      Ionic*      |         N/A         |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| NumPy          |     1.26.4     |     1.24.4     |  1.20.1  |     2.1.3     |      1.24.2      |         N/A         |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Ogre           |                                    1.12.10                                    |         N/A         |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| OpenCV         |     4.6.0      |     3.4.6*     |  4.6.0   |    4.10.0     |      4.6.0       |  4.1.0 / 3.2.0***   |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| OpenSSL        |     3.0.13     |     1.1.1u     |  3.2.2   |    1.1.1w     |      3.0.15      | 1.1.1d / 1.1.1b***  |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Python         |     3.12.3     |     3.8.3      |  3.9.19  |    3.13.0     |      3.11.2      |  3.8.2 / 3.7.5***   |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Qt             |    5.15.13     |    5.12.10     |  5.15.9  |    5.15.16    |      5.15.8      | 5.14.1 / 5.12.5***  |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                 |                                   **Linux only**                                   |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| PCL            |     1.14.0     |      N/A       |  1.12.0  |    1.14.1     |      1.13.0      |       1.10.0        |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                                **RMW DDS Middleware**                                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Cyclone DDS    |                                               0.10.5                                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Fast-DDS       |                                               2.14.4                                                |
```

Comment:
This will likely end up being 3.x, but let's leave it as this until we actually land that support.